### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by AyomipoMajek**
Bookstore: styling done according to microverse requirements which include:

Add any missing JSX elements to your project (without adding extra functionality)
Your final design should match Zeplin's design (this includes all elements + the same styling)
You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/styles/Nav.css [hunk 1]
```diff
@@ -6,6 +6,7 @@
   box-sizing: border-box;
```
Comment (codecaiine) [inline src/styles/Nav.css:6]: - [ ] Your navbar is a bit different from original design . Please make sure that it looks similar to the orignal design 👍

File: src/components/Form.js [hunk 2]
```diff
@@ -34,7 +36,7 @@ function Form() {
           placeholder="Book author"
           onChange={(e) => setbook({ ...book, author: e.target.value })}
         />
-        <button type="submit">ADD BOOK</button>
+        <button className="add" type="submit">ADD BOOK</button>
```
Comment (codecaiine) [inline src/components/Form.js:39]: - [ ] Your form element are not aligned like on the original design . Bookstore CMS make sure to have the same design as defined on the orignal desgin here 👍

File: src/styles/Book.css [hunk 3]
```diff
@@ -50,6 +50,93 @@
   padding-right: 15px;
 }
 
+.completed {
```
Comment (codecaiine) [inline src/styles/Book.css:55]: - [ ] Good job so far 👍  But I think you could improve you design . For this project it's required that your final design match with Zeplin's design (this includes all elements + the same styling). Please fix it in order to make you app looks similar to Zeplin's down ⬇️ 

Bookstore CMS

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @AyomipoMajek , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linters errors ✔️ 
- Good design set up ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @AyomipoMajek , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**